developer/src/kmcmplib/src/Compiler.cpp

```diff
@@ -3516,7 +3517,7 @@ bool UTF16TempFromUTF8(KMX_BYTE* infile, int sz, KMX_BYTE** tempfile, int *sz16)
   try {
     std::wstring_convert<std::codecvt_utf8_utf16<char16_t>, char16_t> converter;
     result = converter.from_bytes((char*)infile, (char*)infile+sz);
-  } catch(std::range_error e) {
+  } catch(std::range_error &e) {
```

Comment:
Not that it matters, but I find this easier to read:
```suggestion
  } catch(std::range_error& e) {
```

Reply:
Okay.